README.md

```diff
@@ -24,6 +24,8 @@ With a mac computer that can run macOS 10.13+, install all the dependencies belo
     * Install with: `xcode-select --install` / [The mac app store](https://itunes.apple.com/us/app/xcode/id497799835)
 * [BUCK for Xcode project generation](https://buckbuild.com/)
 	* [Install instructions](https://buckbuild.com/setup/getting_started.html)
+* [CocoaPods for Xcode project generation](https://cocoapods.org/)
```

Comment:
I would put something that says you only need one of these, unless you want to test project generation for both.